static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -1255,12 +1258,58 @@ function HasDebugIdChecklistItem({
   );
 }
 
+function DebugIdMismatchMessage({
+  debugId,
+  projectSlug,
+}: {
+  debugId: string | null;
+  projectSlug?: string;
+}) {
+  // At this point debugId is always defined. The types need to be fixed
+  if (!debugId) {
+    return (
+      <Fragment>
+        {t(
+          "You already uploaded artifacts with Debug IDs but none of the uploaded source files had a Debug ID matching this stack frame's Debug ID"
+        )}
+      </Fragment>
+    );
+  }
+
+  return tct(
+    "You already uploaded artifacts with Debug IDs but none of the uploaded source files had a Debug ID matching this stack frame's Debug ID: [debugId]",
+    {
+      debugId: projectSlug ? (
+        <LinkButton
+          to={{
+            pathname: `/settings/projects/${projectSlug}/source-maps/`,
+            query: {
+              query: debugId,
+            },
+          }}
+          icon={<IconOpen />}
+          aria-label={t('View source map Debug ID %(debugId)s in project settings', {
+            debugId,
+          })}
```

Comment:
```suggestion
          aria-label={t('View source map Debug ID %(debugId)s in project settings', {
            debugId,
          })}
```
can we just use `tct` here or is `t` preferred?

Reply:
we need to always have a string for aria-labels, and tct returns a React Node